benchmark/sirun/get-results.js

```diff
@@ -0,0 +1,73 @@
+'use strict'
+
+const https = require('https')
+const { execSync } = require('child_process')
+
+const { CIRCLE_TOKEN } = process.env
+
+const ref = process.argv.length > 2 ? process.arv[2] : 'HEAD'
+const gitCommit = execSync(`git rev-parse ${ref}`).toString().trim()
+
+const circleHeaders = {
+  'circle-token': CIRCLE_TOKEN,
```

Comment:
shouldn't need this. since it is a public project you can hit the API no problem.
```suggestion
```

Reply:
I remember having problems when not setting the token, I think what is available and the rate limits are different when not using a token, even for public projects.

Reply:
I was able to run this script without a token without any issues, but it may be true that rate limits might be an issue.

I have a script I use to pull junit xml results from a bunch of builds (I think it is like 50+ API calls), and I haven't hit any limits or issues yet.

Reply:
I changed it so that it will use the token if present, otherwise ignore it.